services/metadactyl-clj/src/metadactyl/routes/apps.clj

```diff
@@ -102,79 +102,79 @@
         :summary "Update an App"
         :description "This service updates a single-step App in the database, as long as the App has not
         been submitted for public use."
-        (service/trap uri apps/update-app current-user (assoc body :id app-id)))
+        (ok (apps/update-app current-user (assoc body :id app-id))))
 
-  (POST* "/:app-id/copy" [:as {uri :uri}]
+  (POST* "/:app-id/copy" []
          :path-params [app-id :- AppIdPathParam]
          :query [params SecuredQueryParamsRequired]
          :return App
          :summary "Make a Copy of an App Available for Editing"
          :description "This service can be used to make a copy of an App in the user's workspace."
-         (service/trap uri apps/copy-app current-user app-id))
+         (ok (apps/copy-app current-user app-id)))
 
-  (GET* "/:app-id/details" [:as {uri :uri}]
+  (GET* "/:app-id/details" []
         :path-params [app-id :- AppIdJobViewPathParam]
         :query [params SecuredQueryParams]
         :return AppDetails
         :summary "Get App Details"
         :description "This service is used by the DE to obtain high-level details about a single App"
-        (service/coerced-trap uri AppDetails apps/get-app-details current-user app-id))
+        (service/coerced-trap nil AppDetails apps/get-app-details current-user app-id))
 
-  (GET* "/:app-id/documentation" [:as {uri :uri}]
+  (GET* "/:app-id/documentation" []
         :path-params [app-id :- AppIdJobViewPathParam]
         :query [params SecuredQueryParams]
         :return AppDocumentation
         :summary "Get App Documentation"
         :description "This service is used by the DE to obtain documentation for a single App"
-        (service/coerced-trap uri AppDocumentation apps/get-app-docs current-user app-id))
+        (service/coerced-trap nil AppDocumentation apps/get-app-docs current-user app-id))
 
-  (PATCH* "/:app-id/documentation" [:as {uri :uri body :body}]
+  (PATCH* "/:app-id/documentation" [:as {body :body}]
```

Comment:
I think you can remove these `body` ones too since that's handled by the binding with the `:body` keyword below. While you're removing stuff already, I figure